### PR TITLE
Feature/#4 sandbox2

### LINKED ISSUE #4: Training typescript and Angular
- Create a sandbox page
- Practice the following by creating samples on the sandbox page 
        1. - Typescript 
        2. - Angular components 
        3. - Angular services
        4. - Angular routing 
        5. - Getting data from an API using http

### PR DESCRIPTION
**Pull request by boukhrismed**
first pull request

### CODE REVIEW

File: API/appsettings.Development.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "BookShare": "Server=localhost; Port=5432; User Id=appuser; Password=Za1987; DataBase=BookShare"
+    "BookShare": "Server=localhost; Port=5432; User Id=appuser; Password=secret; DataBase=BookShare"
```
Comment (ZakariaOB) [inline API/appsettings.Development.json]: To remove

File: client/src/app/_sandbox/dynamic-components/add-banner/add-banner.component.ts [hunk 2]
```diff
@@ -15,28 +15,21 @@ import { AdItem } from './ad-item';
 @Component({
   selector: 'add-banner',
   templateUrl: './add-banner.component.html',
-  styleUrls: ['./add-banner.component.scss'],
+  styleUrls: ['./add-banner.component.css'],
```
Comment (ZakariaOB) [inline client/src/app/_sandbox/dynamic-components/add-banner/add-banner.component.ts]: Please let it : scss

File: client/src/app/_sandbox/dynamic-components/type-ahead/type-ahead.component.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-type-ahead',
+  templateUrl: './type-ahead.component.html',
+  styleUrls: ['./type-ahead.component.scss']
+})
+export class TypeAheadComponent implements OnInit {
```
Comment (ZakariaOB) [inline client/src/app/_sandbox/dynamic-components/type-ahead/type-ahead.component.ts]: Please bshare-typeahead as the kebab case name
For the component name you can let it as : BShareTypeahead

**Review by ZakariaOB — CHANGES_REQUESTED** [review]

File: API/appsettings.Development.json (hunk 1 above)
Comment (boukhrismed) [reply API/appsettings.Development.json]: fixed

File: client/src/app/_sandbox/dynamic-components/add-banner/add-banner.component.ts (hunk 2 above)
Comment (boukhrismed) [reply client/src/app/_sandbox/dynamic-components/add-banner/add-banner.component.ts]: fixed

File: client/src/app/_sandbox/dynamic-components/type-ahead/type-ahead.component.ts (hunk 3 above)
Comment (boukhrismed) [reply client/src/app/_sandbox/dynamic-components/type-ahead/type-ahead.component.ts]: fixed

**Review by ZakariaOB — APPROVED** [review]

**Pull request merged**